edit_form.php

```diff
@@ -98,13 +98,16 @@ public function definition() {
                            get_string('onerow', 'report_customsql'));
 
         $mform->addElement('text', 'emailto', get_string('emailto', 'report_customsql'), 'size = 70');
+        $mform->addElement('text', 'customdir', get_string('customdir', 'report_customsql'), 'size = 70');
```

Comment:
Should this field have a help icon, explaining that the file is saved in CSV format, and a bit about how the file name looks, etc?